### PR TITLE
God clashing fix and Invoke Sevtug nerf

### PR DESCRIPTION
**Pull request by ghost**
- Harvesters now provide 2% success chance to Nar-Sie during clashes with Ratvar.
- Invoke Sevtug, the Formless Pariah has had its duration halved, down to 30 seconds.

### CODE REVIEW

**Comment by Kromgar** [comment]
When are we implenting cult vs cult already?

File: code/game/gamemodes/clock_cult/clock_ratvar.dm [hunk 1]
```diff
@@ -277,6 +277,8 @@
 			shake_camera(M, 4, 3)
 		var/r_success_modifier = (ticker.mode.servants_of_ratvar.len * 2) //2% for each cultist
 		var/n_success_modifier = (ticker.mode.cult.len * 2)
+		for(var/mob/living/simple_animal/hostile/harvester/construct in player_list)
```
Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_ratvar.dm]: I think you messed up the path here.

**Comment by Xhuis** [comment]
@Kromgar I'd like to, but I think that gamemode code is on freeze right now.

**Comment by Xhuis** [comment]
Some people were expressing reasonable concerns about Invoke Sevtug, so I've halved its duration.

**Comment by ChangelingRain** [comment]
Really I'm just going to replace invoke sevtug with some sort of global hallucination scream or something.

**Comment by coiax** [comment]
I like the mind control though.

Why not have a visible message when someone's controlled, thus if you MC the Hos in the middle of the officers, they can tase and cuff him until its over, but if you manage to grab someone alone, they'd be no one to notice the difference.

Also, while we're there, can we add a hud icon or something so servants can see that someone is mindcontrolled? Having to repeatedly say "I'm mind controlled, cuff me." is a little lame.

**Pull request merged**